src/LCT.APICommunications/JfrogApicommunication.cs

```diff
@@ -18,18 +19,21 @@ public abstract class JfrogApicommunication : IJFrogApiCommunication
         protected ArtifactoryCredentials ArtifactoryCredentials { get; set; }
 
         protected string SourceRepoName { get; set; }
+        private static int timeoutSec { get; set; }
```

Comment:
follow the naming convention for variables

Reply:
done